Python/compile.c

```diff
@@ -158,7 +158,8 @@ struct compiler {
     PyFutureFeatures *c_future; /* pointer to module's __future__ */
     PyCompilerFlags *c_flags;
 
-    int c_optimize;              /* optimization level */
+    int c_optimize;              /* optimize bytecode? */
+    int c_optimization_level;    /* optimization level */
```

Comment:
Maybe use `-1` for noopt?

Reply:
-1 has a special meaning for compile functions. I prefer to have a separated field to avoid confusion.